lerna.json

```diff
@@ -0,0 +1,5 @@
+{
+  "packages": ["packages/*"],
+  "npmClient": "npm",
+  "version": "0.0.0"
```

Comment:
Doesn't this version refer to the currently published version? We have releases of the ui-kit, so this needs to be higher right? (see [wix/detox/lerna.json](https://github.com/wix/Detox/blob/master/lerna.json#L13) for reference)